crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1614,6 +1618,9 @@ where
             // Tally of stake for good vs bad responses.
             good_stake: StakeUnit,
             bad_stake: StakeUnit,
+            // If there are conflicting transactions, we note them down and may attempt to retry
+            conflicting_tx_digests:
+                BTreeMap<TransactionDigest, (Vec<(AuthorityName, ObjectRef)>, StakeUnit)>,
```

Comment:
Why do we keep the stake unit? Simply to determine when a quorum has accepted the transaction (and there is thus no need to retry)?